shared/src/commonMain/kotlin/com/example/tasky/agenda/domain/model/AgendaItem.kt

```diff
@@ -37,6 +44,49 @@ data class Event(
     val attendees: List<Attendee>,
     val photos: List<Photo>,
 ) : AgendaItem() {
+    constructor(remoteEvent: RemoteEvent) : this(
+        id = remoteEvent.id,
+        title = remoteEvent.title,
+        description = remoteEvent.description,
+        from = remoteEvent.from,
+        to = remoteEvent.to,
+        remindAt = remoteEvent.remindAt,
+        host = remoteEvent.host,
+        isUserEventCreator = remoteEvent.isUserEventCreator,
+        attendees = remoteEvent.attendees,
+        photos = remoteEvent.photos,
+    )
+
+    fun toUpdateEventBody(
+        deletedPhotoKeys: List<String>,
+        isGoing: Boolean,
+        photos: List<ByteArray>,
+    ): UpdateEventBody =
+        UpdateEventBody(
+            id = id,
+            title = title,
+            description = description,
+            from = from,
+            to = to,
+            remindAt = remindAt,
+            attendeeIds = attendees.map { it.userId },
+            deletedPhotoKeys = deletedPhotoKeys,
+            isGoing = isGoing,
+            photos = photos,
+        )
+
+    fun toCreateEventBody(photos: List<ByteArray>): CreateEventBody =
```

Comment:
I'd define mappers in a separate file since otherwise you're mixing responsibilities here (mapping itself has nothing to do with how the DTO behaves). That way, you have one place for all mappers of one type, since sometimes you need multiple mappers for one class (if you have the same type of model in a local DB for example).
I always define them like this in a separate file:
```
fun YourDto.toYourModel(): YourModel {
    return YourModel(...)
}
```